.gitignore

```diff
@@ -10,6 +10,8 @@
 .coverage
 .coverage.*
 __pycache__/
+**/.mypy_cache/
```

Comment:
this should not be needed, `.ruff_cache/` and `.mypy_cache/` come with a `.gitignore` inside